### DISCUSSION TITLE
일정 관련

### DISCUSSION THREAD

**Discussion by ffdd270** [juntz/ZtmyCard#22] [General]
금주 + 차주 초까지 제가 바쁜 일이 있어서 진행이 꽤 더딜 것 같습니다. 혹시 우선 순위를 배정해주시면 그에 맞춰서 수정해놓겠습니다 'ㅁ 'b( 제 생각에는 샌드박스 모드가 꽤 높을 것 같습니다 )

**Comment by juntz** [comment]
멀티플레이를 강제하는 현재 프로그램 구조상 샌드박스 모드는 구현이 어려울 것 같습니다.
대신 멀티플레이에 디버그 패널을 추가하는 이슈(#26)를 생성했으니 그쪽을 먼저 봐 주시면 감사하겠습니다.
이외에 다른 이슈들은 특별히 우선순위가 있지는 않을 것 같으니 편하신 대로 작업하시면 될 것 같네요.
항상 기여해주셔서 감사합니다. 🙏
